_posts/2023-03-09-group-equivariant-cnn-1/group-equivariant-cnn-1.Rmd

```diff
@@ -0,0 +1,148 @@
+
+### Groups from symmetries
+
+Here is a square.
+
+```{r, echo=FALSE, eval=TRUE, fig.alt = "A square in its default position, aligned horizontally to a virtual (invisible) x-axis."}
+knitr::include_graphics("images/square1.png")
+```
+
+Now close your eyes.
+
+Now look again. Did something happen to the square?
+
+
+```{r, echo=FALSE, eval=TRUE, fig.alt = "A square in its default position, aligned horizontally to a virtual (invisible) x-axis."}
+knitr::include_graphics("images/square1.png")
+```
+
+You can't tell. Maybe it was rotated; maybe it was not. On the other hand, what if the vertices were numbered?
+
+
+```{r, echo=FALSE, eval=TRUE, fig.alt = "A square in its default position, with vertices numbered from 1 to 4, starting in the lower right corner and counting ant-clockwise."}
+knitr::include_graphics("images/square2.png")
+```
+
+Now you'd know.
+
+Without the numbering, could I have rotated the square in any way I wanted? Evidently not. This would not go through unnoticed:
+
+
+```{r, echo=FALSE, eval=TRUE, fig.alt = "A square, rotated anti-clockwise by a few degrees."}
+knitr::include_graphics("images/square3.png")
+```
+
+There are exactly four ways I could have rotated the square without raising suspicion. Those ways can be referred to in different ways; one simple way is by degree of rotation: 90, 180, or 270 degrees. Why not more? Any further addition of 90 degrees would result in a configuration we've already seen.
+
+
+```{r, echo=FALSE, eval=TRUE, fig.alt = "Four squares, with numbered vertices each. The first has vertex 1 on the lower right, the second one rotation up, on the upper right, and so on."}
+knitr::include_graphics("images/square4.png")
+```
+
+The above picture shows three squares, but I've listed three possible rotations. What about the situation on the left, the one I've taken as an initial state? It could be reached by rotating 360 degrees (or twice that, or thrice, or ...) But the way this is handled, in math, is by treating it as some sort of "null rotation", analogously to how $0$ acts in addition, $1$ in multiplication, or the identity matrix, in linear algebra.
+
+Altogether, we thus have four *actions* that could be performed on the square (an un-numbered square!) that would leave it as-is, or *invariant*. These are called the *symmetries* of the square. A symmetry, in math/physics, is a quantity that remains the same no matter what happens as time evolves. And this is where *groups* come in: It's them -- concretely, their *elements* -- that effectuate actions like rotation.
```

Comment:
```suggestion
Altogether, we thus have four *actions* that could be performed on the square (an un-numbered square!) that would leave it as-is, or *invariant*. These are called the *symmetries* of the square. A symmetry, in math/physics, is a quantity that remains the same no matter what happens as time evolves. And this is where *groups* come in. Groups -- concretely, their *elements* -- effectuate actions like rotation.
```

Reply:
^^ changed because _them_ would more properly/grammatically be _they_ since you're saying "They…effectuate actions” (like her vs. she), but "It is they" sounds super weird.

Reply:
Oh, I see ... I'd like to have something with that kind of syntax ("cleft sentence" it's called I'm told), but if it turns out ungrammatical ... Maybe I'll still have an alternative idea later, committing ...